components/dash-core-components/src/components/RadioItems.react.js

```diff
@@ -63,39 +74,48 @@ export default class RadioItems extends Component {
 }
 
 RadioItems.propTypes = {
+    /**
+     * An array of options, or inline dictionary of options
+     */
+    options: PropTypes.oneOfType([
+        /**
+         * Simpler `options` representation in dictionary format
+         * {`label1`: `value1`, `label2`: `value2`, ... }
```

Comment:
I believe that `value` needs to be unique but label doesn't need to be unique, so this should be inverted: `{'value1': 'label1', 'value2': 'label2'}`

Reply:
@chriddyp @jackparmer @akksi
Two things here,
1. Unfortunately, I haven't found a way to iterate key as its own type, but all of Object methods (`entries`, `keys`, `for in`, etc) just gives string type key.
I think idea is good but we won't be dealing non-string `value` properly.

2. Also, I'm removing the `PropTypes.oneOfType([string, number, bool])` constraint because we'll be actually trying to validate the `label` type which is not what we really wanted.
https://plotly.slack.com/archives/C01HT8G2RHQ/p1631628303425800?thread_ts=1631542387.390300&cid=C01HT8G2RHQ

https://github.com/workaholicpanda/dash/pull/6

Reply:
That's correct - `values` will only be able to be strings. That's OK - It's just a limitation of this data shape. Users won't necessarily see this in Python as Python will serialize the keys to strings.